pycqed/measurement/pulse_sequences/multi_qubit_tek_seq_elts.py

```diff
@@ -844,6 +875,229 @@ def parity_single_round_seq(ancilla_qubit_name, data_qubit_names, CZ_map,
     return seq, np.arange(seq.n_acq_elements())
 
 
+def multi_parity_multi_round_seq(ancilla_qubit_names,
```

Comment:
@nathlacroix Maybe you could update this function to use the new repeat pattern syntax. This should be the only place where we use it on XLD.

Reply:
@antr not sure I fully understand which channels which pulses should be repeated here, since it is a bit hacky ;) I would rather have Christian do it. The doc string of the sequence.repeat() is quite straight forward.

Reply:
ok, we need to make Christian make a github account then 😆